packages/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode, useState } from 'react'
+
+import { Divider, Item, List } from './styled'
+
+interface AccordionItemProps {
+  children?: ReactNode
+  className?: string
+}
+
+export default List
+export { List as Accordion }
+
+/* istanbul ignore next */
+export function useAccordionController() {
+  const [activeItem, setActiveItem] = useState<number | null>()
```

Comment:
Should we use `null` as the initial state value here, or is that the default value when no argument is passed?
```suggestion
  const [activeItem, setActiveItem] = useState<number | null>(null)
```